test/stringClassTest.js

```diff
@@ -0,0 +1,122 @@
+const expect = require('chai').expect;
+require('../src/stringClass');
+
+describe('StringManipulation Class', () => {
+  describe('hasVowels', () => {
+    it('should return "true" if word has vowels', () => {
+      expect('andela'.hasVowels()).to.be.true;
+    });
+    it('should also return "true" regardless of the vowel case', () => {
+      expect('clAss'.hasVowels()).to.be.true;
+    });
+    it('should return "false" if word has only consonants', () => {
+      expect('hymn'.hasVowels()).to.be.false;
+    });
+  });
+
+  describe('toUpper', () => {
+    it('should return the string in uppercase', () => {
+      expect('amity'.toUpper()).to.equal('AMITY');
+      expect('AnDelA'.toUpper()).to.equal('ANDELA');
+    });
+  });
+
+  describe('toLower', () => {
+    it('should return the string in lower case', () => {
+      expect('AnDeLA'.toLower()).to.equal('andela');
+    });
+  });
+
+  describe('ucFirst', () => {
+    it('should return the string with the first letter in upper case', () => {
+      expect('omoleye'.ucFirst()).to.equal('Omoleye');
```

Comment:
It's best to add more test cases just to be thorough. e.g What happens when the first letter is already uppercase?.